code/game/objects/effects/contraband.dm

```diff
@@ -688,7 +688,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/official/random, 32)
 
 /obj/structure/sign/poster/official/no_erp
 	name = "No ERP"
-	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
+	desc = "This poster reminds the crew that Eroticism and Pornography are banned on Daedalus stations."
```

Comment:
This was removed because
A. No shit sherlock.
B. It was kind of shocking to see? I don't want mentions of it in my codebase.